modules/common/GoogleBucketResource.py

```diff
@@ -83,26 +74,24 @@ def get_bucket(self):
         except google.cloud.exceptions.NotFound:
             logger.error("Sorry, that bucket {} does not exist!".format(self.bucket_name))
             return None
-        except exceptions.Forbidden as ef:
+        except exceptions.Forbidden:
             logger.error(" ERROR: GCS forbidden access, path={}".format(self.bucket_name))
             return None
         return bucket
 
-
-
-    #google_resource.list_blobs('es5-sufentanil/tmp/','/', None, None)
+    # google_resource.list_blobs('es5-sufentanil/tmp/','/', None, None)
```

Comment:
Commented out code can be removed.

Reply:
I clean a little bit the code. Removed unused code